.github/ISSUE_TEMPLATE/design.md

```diff
@@ -8,5 +8,16 @@ labels: 🗺️ design
 
 _This issue tracks design work through the [CDH design review workflow](https://princeton-cdh.github.io/docs/workflows/#workflow-5)._
 
+Here is a link to the  
+
```

Comment:
what would this be a link to?

Reply:
I had the same question! I wondered if this was unfinished.

@gissoo maybe you could put a placeholder for what you expect to go here?

Reply:
@rlskoeser @thatbudakguy I didn't put anything on purpose, is it ok if it's empty? It would be easier for me ...I don't want to put "designs" because I will have to delete it every time ...I rarely say "designs", it would be better if it's empty if ok.

Reply:
Fine by me if you don't need a prompt or placeholder for whatever goes there.  (If we expected anyone else to use the template we might need to revisit).